CHANGELOG.md

```diff
@@ -1,9 +1,6 @@
 # CHANGELOG
-## v4.4.0-2
```

Comment:
why remove 4402?

Reply:
is it because 4402 is still pre-release: https://github.com/Manta-Network/Manta/releases/tag/v4.4.0-2 can we release it now?

Reply:
yeah, we can do all those of changes in one runtime upgrade

Reply:
the changelog tool couldn't handle the 4.4.0-2 format. 

I suppose it is fine to put the releases together